### PR TITLE
Enhance configurability

### PR DESCRIPTION
**Pull request by tomquisel**
- move to the `.env` file for some configuration
- Make the scraper more configurable
- make the `README.md` run command work

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -9,31 +9,28 @@ services:
...
 #      - ./db/data/:/var/lib/postgresql/data
     environment:
-      POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: 1234  # To be moved to environment file
-      POSTGRES_DATABASE: clinical_trials_status
-    expose:
-      - 5432
-    ports:
-      - 5432:5432
-    networks:
```
Comment (dstuck) [inline docker-compose.yml:19]: Oh, so I hadn't included this network in the scraper which is why it couldn't connect. Makes sense to remove it in a build as simple as this

File: docker-compose.yml [hunk 2]
```diff
@@ -9,31 +9,28 @@ services:
...
     links:
       - db
-    networks:
-      - psql-pgql
-networks:
-  psql-pgql:
+  scraper:
+    container_name: scraper
+    # image: dstuck/clinical-trials-api-scraper:latest
+    build: ../clinical-trials-api-scraper
```
Comment (dstuck) [inline docker-compose.yml:40]: so is this using a local build now for testing? That's a good idea

**Review by dstuck — APPROVED** [review]
Looks good!

File: docker-compose.yml (hunk 2 above)
Comment (tomquisel) [reply docker-compose.yml:40]: Yeah! It was honestly accidental, something I set up for local dev. If you're happy with it though, sounds good!

**Pull request closed without merging**